webofneeds/won-core/src/main/java/won/protocol/model/Connection.java

```diff
@@ -33,6 +33,10 @@
   @Column( name = "id" )
   private Long id;
 
+  @Version
```

Comment:
changes that affect the database must now (after integration of flyway pull request) also provide a new flyway migration script => this must be added if this pull request is integrated after flyway pull request